### ISSUE TITLE
Change Plot optiones in ManagedDockWindow

### ISSUE THREAD

**Issue by MiguelBL248** [pymeasure/pymeasure#1073]
Hello everyone,

I am using ManagedDockWindow in order to plot two pair of values (pressure_1 and pressure_2) as a function of the iterations. My question is how I can change plot aspects such as the size of the numbers, among many others. 

Moreover, my final goal is to plot the values of the pressure as a function of the time (hh:mm:ss), but when plotting I  have irregular spaces between points (example: 13:59:59 to 14:00:00, 135959 is not the number before 140000, so there is a big space between points). I saw that in matplotlib i can use the method plt.xticks to solve this, but I am not sure how to do it in the managed dock window case.

To sum up, my question is how I can change plot options in managed dock window.

Thanks in advance.
